fixture-monkey-api/src/main/java/com/navercorp/fixturemonkey/api/property/SingleElementProperty.java

```diff
@@ -0,0 +1,155 @@
+/*
+ * Fixture Monkey
+ *
+ * Copyright (c) 2021-present NAVER Corp.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.navercorp.fixturemonkey.api.property;
+
+import java.lang.annotation.Annotation;
+import java.lang.reflect.AnnotatedType;
+import java.lang.reflect.Type;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+import java.util.Optional;
+import java.util.OptionalDouble;
+import java.util.OptionalInt;
+import java.util.OptionalLong;
+import java.util.function.Function;
+import java.util.function.Supplier;
+import java.util.stream.Collectors;
+
+import javax.annotation.Nullable;
+
+import com.navercorp.fixturemonkey.api.type.Types;
+
+public final class SingleElementProperty implements Property {
```

Comment:
`@API(since = "1.0.16", status = Status.EXPERIMENTAL)` is needed.

Reply:
[9edca18](https://github.com/naver/fixture-monkey/pull/951/commits/9edca1840cf2c7c1bf404bf8224e563c604b40c6)
[c827311](https://github.com/naver/fixture-monkey/pull/951/commits/c82731111a1f39edee8f5fe627d7e0247eef8a6e)